src/nopoll_decl.h

```diff
@@ -168,13 +168,17 @@
 /* additional includes for the windows platform */
 
 /* _WIN32_WINNT note: If the application including the header defines
- * the _WIN32_WINNT, it must include the bit defined by the value
- * 0x400. */
+ * the _WIN32_WINNT, it must include the bits defined by the value
+ * 0x501. */
```

Comment:
As I replied in the other review, we always do cross-compilation from Linux. If forcing _WIN32_WINNT to 0x501 bothers you, we do not mind keeping it as 0x400.